### PR TITLE
lists-feat

### PR DESCRIPTION
**Pull request by Karan-Bharti**
added different types of lists
image
image
image

### CODE REVIEW

File: lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.list-spaced,
+.list-stacked,
+.list-stacked-borders{
+    display: flex;
+    gap: 2rem;
+}
+.list-item{
+list-style-type: none;
+font-weight: bold;
```
Comment (sakleprajwal) [inline lists/lists.css:9]: Give proper and consistent indentation

File: lists/lists.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                <a href="/Alert/alert.html" class="drawer-alert"> Alert</a>
+                <a href="/Avatar/avatar.html" class="drawer-avatar">Avatar</a>
+                <a href="" class="drawer-badge">Badges</a>
+                <a href="" class="drawer-button">Button</a>
+                <a href="" class="drawer-card">Card</a>
+                <a href="" class="drawer-image">Image</a>
+                <a href="" class="drawer-input">Input</a>
+                <a href="" class="drawer-grid">Grid</a>
+                <a href="" class="drawer-snackbar">Snackbar</a>
+                <a href="" class="drawer-lists">Lists</a>
```
Comment (sakleprajwal) [inline lists/lists.html:34]: Use # in href instead of blank.

**Review by sakleprajwal — APPROVED** [review]
Suggested some minor changes.

File: lists/lists.html (hunk 2 above)
Comment (Karan-Bharti) [reply lists/lists.html:34]: soon i will upload all the links

**Review by raman1999 — APPROVED** [review]
looks good to me

**Pull request merged**